src/components/PostPageDetail/index.js

```diff
@@ -67,9 +67,9 @@ const PostPageDetailIdComponent = (props) => {
   const [statusUpvote, setStatusUpvote] = React.useState(false);
   const [statusDownvote, setStatusDowvote] = React.useState(false);
   const [loadingPost, setLoadingPost] = React.useState(false);
-  const [item, setItem] = React.useState(null);
   const navigation = useNavigation();
   const route = useRoute();
+  const [item, setItem] = React.useState(route?.params?.data);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from initializing `item` with `null` to `route?.params?.data` is a good one as it allows the component to start with some initial data if available. However, please ensure that `route.params.data` is always defined and in the correct format before this component is rendered. If there's a chance that it might not be, you should add error handling for that case.
```diff
- const [item, setItem] = React.useState(null);
+ const [item, setItem] = React.useState(route?.params?.data);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->